src/Containers/FileUpload/FileUpload.tsx

```diff
@@ -584,71 +508,92 @@ export const FileUpload: React.FC<IFileUploadProps> = ({
-            >
-                <SubContainer
-                    minHeight={minHeight}
-                    minWidth={minWidth}
-                    disabled={disabled}
+            <Dropzone multiple>
+                {() => (
+                    <Container
+                        {...getRootProps({
+                            dashed: true,
+                            withFlexCenter: true,
+                            withBorder: !state.isDragEnter,
+                            isDragEnter: state.isDragEnter,
+                            padding: '10px',
+                            margin: `${MARGIN}px`,
+                        })}
+                        transitionDuration={animationDuration}
+                    >
+                        <SubContainer
+                            minHeight={minHeight}
+                            minWidth={minWidth}
+                            disabled={isDisabled ?? false}
+                        >
+                            {state.isDragEnter ? (
+                                <FileMovingAnimation />
+                            ) : (
+                                <Icon as={Image} width={140} height={80} />
+                            )}
+                            <TextLayout bold color="DarkBlue">
+                                {title}
+                            </TextLayout>
+                            <TextLayout
+                                size="small"
+                                bold
+                                color="rgba(128,128,128,.8)"
+                            >
+                                {subTitle}
+                            </TextLayout>
+                            <input {...getInputProps()} />
+                        </SubContainer>
+                    </Container>
+                )}
+            </Dropzone>
+            {informativePanelsState.values.map((value, index) => (
+                <BottomPanel
+                    key={index} //eslint-disable-line
+                    isUploading={value.isUploading.value}
+                    onCancelUploading={onCancelUploading(value)}
+                    withBorder
+                    padding={
+                        value.isFailure.value || value.isSuccess.value
+                            ? undefined
+                            : '30px 20px 43px 20px'
+                    }
+                    opacity={
+                        value.isUploading.opacity ||
+                        value.isSuccess.opacity ||
+                        value.isFailure.opacity
+                    }
+                    position={
+                        value.isUploading.isPosition &&
+                        value.isFailure.isPosition &&
+                        value.isSuccess.isPosition
+                    }
+                    ref={loadingContainerRef}
+                    overflow="hidden"
+                    width={
+                        value.isFailure.value || value.isSuccess.value
```

Comment:
This code is duplicated in the next few lines why not make a function called

IsSuccessOrError?